apps/sentry/src/inngest/functions/users/sync-users-page.ts

```diff
@@ -1,96 +1,115 @@
-import type { User } from '@elba-security/sdk';
-import { Elba } from '@elba-security/sdk';
-import { eq } from 'drizzle-orm';
+import { Elba, type User } from '@elba-security/sdk';
 import { NonRetriableError } from 'inngest';
-import { type MySaasUser, getUsers } from '@/connectors/users';
+import { eq } from 'drizzle-orm';
+import { type SentryUser, getUsers} from '@/connectors/users';
 import { db } from '@/database/client';
 import { Organisation } from '@/database/schema';
 import { env } from '@/env';
 import { inngest } from '@/inngest/client';
-import { decrypt } from '@/common/crypto';
 
-const formatElbaUser = (user: MySaasUser): User => ({
-  id: user.id,
-  displayName: user.username,
-  email: user.email,
+
+const formatElbaUser = (user: SentryUser): User => ({
+  id: user.user.id,
+  displayName: user.user.name,
+  email: user.user.email,
+  role: user.role,
+  authMethod: undefined,
```

Comment:
when we access has2fa, then check user.has2fa? 'mfa' : 'password'

Reply:
Done